smithy-swift-codegen/src/test/kotlin/WaiterGeneratorTests.kt

```diff
@@ -0,0 +1,61 @@
+import io.kotest.matchers.string.shouldContainOnlyOnce
+import org.junit.jupiter.api.Test
+import software.amazon.smithy.codegen.core.SymbolProvider
+import software.amazon.smithy.model.Model
+import software.amazon.smithy.swift.codegen.SwiftSettings
+import software.amazon.smithy.swift.codegen.WaiterGenerator
+import software.amazon.smithy.swift.codegen.core.CodegenContext
+import software.amazon.smithy.swift.codegen.integration.ProtocolGenerator
+import software.amazon.smithy.swift.codegen.integration.SwiftIntegration
+
+class WaiterGeneratorTests {
+
+    @Test
+    fun testRenderTestWaiter() {
```

Comment:
Can we add a test that verifies operations without the `waitable` trait, do not render a waiter?

Reply:
Added